bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -14,10 +16,8 @@
 logger = logging.getLogger(__name__)
 
 
-ANNOYED_EXPRESSIONS = ["-_-", "-.-"]
-
 WRONG_ANS_RESPONSE = [
-    "No one gave the correct answer",
+    "No answered correctly!",
```

Comment:
Looks like there's a word missing here.
```suggestion
    "No one answered correctly!",
```

Reply:
got it